samples/Samples.FakeKudu/Properties/launchSettings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "profiles": {
+    "Samples.FakeKudu": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "COR_ENABLE_PROFILING": "1",
+        "COR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
+        "COR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\Datadog.Trace.ClrProfiler.Native.dll",
+
+        "CORECLR_ENABLE_PROFILING": "1",
+        "CORECLR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
+        "CORECLR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\Datadog.Trace.ClrProfiler.Native.dll",
+
+        "DD_DOTNET_TRACER_HOME": "$(ProjectDir)$(OutputPath)profiler-lib",
+        "DD_INTEGRATIONS": "$(ProjectDir)$(OutputPath)profiler-lib\\integrations.json",
+        "DD_AZURE_APP_SERVICES": "1", // We need to pretend we are in the context of AAS to filter by the APP_POOL_ID
```

Comment:
nit: It would be nice to have more consistency in all of the Azure App Service variables. Perhaps they would all have the prefix `DD_AAS_XXXX`

Reply:
Didn't we decide we wanted to be able to use the trace agent variables outside of AAS contexts?

Reply:
Right, you can disregard this now